plop-templates/component-pkg/index.ts.hbs

```diff
@@ -0,0 +1,2 @@
+// export * from "./use{{capitalize componentName}}"
```

Comment:
Remove commented line?

Reply:
I think I want that left there for if we use Hooks... without the line commented we get a typescript error and it's easy to cleanup after it's generated